tests/baselines/reference/isolated-declarations/original/diff/ES5SymbolProperty1.d.ts.diff

```
@@ -0,0 +1,35 @@
+// [[Reason: undefined]] ////

```

Probably the tests need to fail if this is `undefined`.

---

Actually, is this temporary? I noticed that this is in a subdir so maybe this is still leftover.

---

Yes. This should fail if this is undefined. I just had not gotten around to categorizing all the tests yet.